### DISCUSSION TITLE
Video File Not Found

### DISCUSSION THREAD

**Discussion by jcostel2** [biigle/core#534] [Q&A]
Hi there, I uploaded videos from MS Azure to BIIGLE but not all of the videos will play. When I select to annotate a video that has no image, I get an error that says that the video is not found. Is there a size limit of videos when adding them to BIIGLE? 

![image](https://user-images.githubusercontent.com/122479023/212079522-3a5047f0-db8d-4495-9033-1e033fe177cb.png)
![image](https://user-images.githubusercontent.com/122479023/212079627-413704ac-1e2b-4675-817e-8e04b2f54605.png)

All videos work properly in MS Azure and I have deleted from BIIGLE and tried to re-upload them but it did not work. Any help with this issue would be greatly appreciated. 
Thanks!

**Comment by mzur** [comment]
I checked with one of the videos and the network connection between the BIIGLE servers and your storage location seems to be extremely slow. Our server takes a very long time to load the video. Sometimes no data is transferred at all and BIIGLE will cancel the operation if this time exceeds 10 s. So that's why some of your videos weren't processed.

Where are the videos stored physically? Maybe this can be improved by moving the files a little closer to Germany. Alternatively, you can always use the file upload of BIIGLE and store the files on our servers.

**Reply by jcostel2** [reply]
I am unable to move the files closer to Germany, I did try to upload the files to the BIIGLE server and go from there because that seems like the best option but I keep getting an error saying that "the server did not respond". Any ideas what could be causing that?

**Reply by mzur** [reply]
The logfiles indicate that you use Edge. Can you please try with Chrome or Firefox?

**Reply by jcostel2** [reply]
I have tried with Chrome and I am getting the same error

**Reply by dlangenk** [reply]
Do you get that error directly or after several minutes? I got that error after my pc went to standby and I woke it up again. However if you click submit again it did not start anew but at the point it got the error.

**Reply by jcostel2** [reply]
Sometimes I am getting the error directly and other times it is coming up after several minutes

**Reply by mzur** [reply]
I assume that this is caused by the same slow/unreliable internet connection. How fast can you upload the files (just an estimate, how long does it take to upload a file and how big is the file)?

Maybe it's a temporary issue. I haven't heard from anyone else from your institute who has these issues.

I'll try to rerun the video processing a few times. I think BIIGLE managed to process a few additional videos since the last time I did that. Maybe we can get all videos processed eventually. If the connection issue persists, this will cause problems with Largo and video object tracking, however. Annotation should work fine, though (once #532 is solved).

**Reply by jcostel2** [reply]
It is taking the file approximately 2 minutes to upload and the size is roughly 2.18 GB. I can wait a little bit and try again later to see if it is a temporary issue.

**Reply by mzur** [reply]
That's quite fast, actually. Nothing like what I see when I try to download a file from Azure on our end. Could you please open the browser developer tools (<kbd>F12</kbd>), select the network tab and then try to upload files to the storage request again? Any error should show up in the dev tools and you could send me a screenshot. You can also send the screenshot to <info@biigle.de> if it might contain anything confidential.

**Reply by barbaraneves1** [reply]
Hello, I'm having the same issue (in Canada). I wonder if the issue was resolved for @jcostel2 ? Thank you,

**Reply by mzur** [reply]
The issue was not resolved (at least on our end). @barbaraneves1 do you see similar error messages if you want to upload files to BIIGLE?

**Reply by barbaraneves1** [reply]
No error messages at the time of uploading. However, when I click on the video immediately after uploading it, the message is that the video is not yet ready, come back later (I think this is normal, though?). I did notice that if I delete a video that is not working, and re-upload it, sometimes it works. But not always. I should also say that even for videos that are working, we started to have issues with largo, as you predicted. Some images just don't show, even if annotation was successful. Anyway, wondering if this is a bandwidth issue, or video size, or MS Azure (our current platform). I was also curious about whether @jcostel2 had made it work. Thank you all.

**Reply by jcostel2** [reply]
@barbaraneves1 Hi, the way that I had to make this issue work was, instead of uploading the videos from MS Azure we uploaded the videos to the Biigle platform itself and then added them to the project. It took a longer amount of time but it has worked so far. Hope this helps!

**Reply by barbaraneves1** [reply]
ok, that helps, thank you. I'm inquiring with other colleagues on whether they are also having this issue (looking more Azure-related than something else?), because uploading the videos to the BIIGLE platform could work, but it wouldn't be sustainable for us :(

**Reply by mzur** [reply]
@barbaraneves1 Feel free to write me at info@biigle.de so we can try to do some targeted tests to find what the problem is.
